src/routes/trip.route.js

```diff
@@ -16,5 +16,6 @@ router.post('/multi-city', authMiddleware.checkUserLoggedIn, authMiddleware.chec
 router.get('/requests', authMiddleware.checkUserLoggedIn, requestValidation, TripController.userTripRequestList); // user request list route
 router.get('/locations', authMiddleware.checkUserLoggedIn, TripController.viewAvailableLocations);
 router.post('/requests/:tripId/comments', authMiddleware.checkUserLoggedIn, UserValidation.validateUserComment, CommentController.addCommentOnTripRequest); // user comment on request trip route
+router.get('/request/comments', authMiddleware.checkUserLoggedIn, UserValidation.validateViewComment, CommentController.viewCommentPostedOnTripRequest); // user view comment posted on the thread
```

Comment:
@erickyvand This route should indicate the trip-request-id whose comments should be retrieved.